### PR TITLE
Insert columns (CU-86du6jp33)

### PR DESCRIPTION
**Pull request by elanawood**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86du6jp33 Add "Organization", "Funder", "Zip/Postal Code" to Clients table. Fill in "Building Type"](https://app.clickup.com/t/86du6jp33)

File: server/db/050-insertColumns.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    -- The migration ID of this file
+    migrationId := 50;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE clients ADD COLUMN Organization text ;
```
Comment (micahdbak) [inline server/db/050-insertColumns.sql:14]: Please add these columns to the `clients_extension` table, not `clients`

++ Please label them in all lowercase like so:

organization
funder
postal_code

to match the other columns!

Comment (elanawood) [reply server/db/050-insertColumns.sql:14]: Ok! Changes made!

Comment (elanawood) [reply server/db/050-insertColumns.sql]: done!

**Review by micahdbak — APPROVED** [review]
Looks good 👍

**Pull request merged**